lib/ecto/parameterized_type.ex

```diff
@@ -100,7 +100,7 @@ defmodule Ecto.ParameterizedType do
   @callback init(opts :: opts()) :: params()
 
   @callback change(old_value :: any(), new_value :: any(), params :: params()) ::
-              {:ok, new_value :: any()} | {:error, keyword()} | :skip
+              {:ok, new_value :: any(), valid? :: boolean()} | {:ok, new_value :: any()} | {:error, keyword()} | :skip
```

Comment:
```suggestion
              {:ok, new_value :: any(), valid? :: boolean()} | {:error, keyword()} | :skip
```

Reply:
@josevalim Why do you want to do that? The `valid?` was added only for `:embeds` purpose whereas other types still should return `{:ok, new_value}`, right?

Reply:
The other types can return `{:ok, new_value, true}`. I think it is easier for everyone if the contracts are as minimal as they can be, instead of allowing many different returns.

Reply:
ah, so the purpose is to keep contracts minimal - no problem :slightly_smiling_face: